metaissue_test.go

```diff
@@ -378,6 +378,351 @@ func TestIssueService_GetCreateMeta_Success(t *testing.T) {
+					"32x32": "https://my.jira.com/secure/projectavatar?size=medium&pid=11300&avatarId=14405"
+				},
+				"projectCategory": {
+					"self": "https://my.jira.com/rest/api/2/projectCategory/10100",
+					"id": "10100",
+					"description": "",
+					"name": "Product & Development"
+				}
+			}]
+		},
+		"assignee": {
+			"required": true,
+			"schema": {
+				"type": "user",
+				"system": "assignee"
+			},
+			"name": "Assignee",
+			"autoCompleteUrl": "https://my.jira.com/rest/api/latest/user/assignable/search?issueKey=null&username=",
+			"hasDefaultValue": true,
+			"operations": [
+				"set"
+			]
+		},
+		"priority": {
+			"required": false,
+			"schema": {
+				"type": "priority",
+				"system": "priority"
+			},
+			"name": "Priority",
+			"hasDefaultValue": true,
+			"operations": [
+				"set"
+			],
+			"allowedValues": [{
+				"self": "https://my.jira.com/rest/api/2/priority/1",
+				"iconUrl": "https://my.jira.com/images/icons/priorities/blocker.svg",
+				"name": "Immediate",
+				"id": "1"
+			}, {
+				"self": "https://my.jira.com/rest/api/2/priority/2",
+				"iconUrl": "https://my.jira.com/images/icons/priorities/critical.svg",
+				"name": "Urgent",
+				"id": "2"
+			}, {
+				"self": "https://my.jira.com/rest/api/2/priority/3",
+				"iconUrl": "https://my.jira.com/images/icons/priorities/major.svg",
+				"name": "High",
+				"id": "3"
+			}, {
+				"self": "https://my.jira.com/rest/api/2/priority/6",
+				"iconUrl": "https://my.jira.com/images/icons/priorities/moderate.svg",
+				"name": "Moderate",
+				"id": "6"
+			}, {
+				"self": "https://my.jira.com/rest/api/2/priority/4",
+				"iconUrl": "https://my.jira.com/images/icons/priorities/minor.svg",
+				"name": "Normal",
+				"id": "4"
+			}, {
+				"self": "https://my.jira.com/rest/api/2/priority/5",
+				"iconUrl": "https://my.jira.com/images/icons/priorities/trivial.svg",
+				"name": "Low",
+				"id": "5"
+			}]
+		},
+		"labels": {
+			"required": false,
+			"schema": {
+				"type": "array",
+				"items": "string",
+				"system": "labels"
+			},
+			"name": "Labels",
+			"autoCompleteUrl": "https://my.jira.com/rest/api/1.0/labels/suggest?query=",
+			"hasDefaultValue": false,
+			"operations": [
+				"add",
+				"set",
+				"remove"
+			]
+		}
+	}
+	}`)
+	})
+
+	editMeta, _, err := testClient.Issue.GetEditMeta(&Issue{Key: "PROJ-9001"})
+	if err != nil {
+		t.Errorf("Expected nil error but got %s", err)
+	}
+
+	requiredFields := 0
+	fields := editMeta.Fields
+	for _, value := range fields {
+		for key, value := range value.(map[string]interface{}) {
+			if key == "required" && value == true {
+				requiredFields = requiredFields + 1
+			}
+		}
+
+	}
+	if requiredFields != 5 {
+		t.Errorf("Expected 5 required fields from Create Meta information, got %d", requiredFields)
+	}
+}
+
+func TestIssueService_GetEditMeta_Fail(t *testing.T) {
+	_, _, err := testClient.Issue.GetEditMeta(&Issue{Key: "PROJ-9001"})
+	if err == nil {
+		t.Error("Expected non nil err, received nil")
```

Comment:
I had to re-read this error a couple times to ensure I understood what was going on. I think this is due to saying `non nil err`. Maybe this could be reworded as:

```
Expected to receive an error, but received `nil` instead
```

In addition, can you assert on the what kind of error you expect to see?
https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully